azerbaijani/nağıl

```diff
@@ -0,0 +1,7 @@
+Biri var idi, biri yox idi. Bir ayı, günü keçmiş kişi var idi. Bir arvadı var idi. Bunların yeddi oğlanları var idi. Amma heç qız uşaqları yox idi. Yeddi qardaşlar bir bacıları olmadığından çox qəmgin idilər. Bir gün yeddi qardaşlar analarına dedilər:
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.